### PR TITLE
Address part of Adriana's design feedback

### PR DESCRIPTION
**Pull request by experimatt**
- [x] Add twitter
- [x] Add text shadow to welcome section
- [x] Add `.` in `St. Paul`
- [x] Add favicon
- [x] Show menu in navbar if on desktop & dropdown if mobile (screenshots below)

I don't think the icons we're using now are perfect, and we can certainly make the tooltip color match our branding better, but those are minor details we can sort out later.

screen shot 2018-05-30 at 9 40 03 pm
screen shot 2018-05-30 at 9 40 35 pm

### CODE REVIEW

**Comment by experimatt** [comment]
I don't think the icons we're using now are perfect, and I think we can change the color of the tooltip so it matches our branding better, but I think we can sort those minor details out later.

**Comment by experimatt** [comment]
@jenessawhite on it!

**Comment by camckin10** [comment]
Where did you add Twitter?

File: src/components/Contact.js [hunk 1]
```diff
@@ -51,14 +51,38 @@ const Contact = () => {
...
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          email us
+        </a>.
+      </p>
+      <hr />
+      <h3>Twitter</h3>
+      <p>
+        Find us on twitter at{' '}
```
Comment (experimatt) [inline src/components/Contact.js]: @camckin10 this is where I added twitter -- in the contact section

**Comment by experimatt** [comment]
@jenessawhite how's this look?
screen shot 2018-05-31 at 9 02 30 pm

**Comment by jenessawhite** [comment]
@experimatt much better.

**Pull request merged**